Resources/Documentation/settings.md

```diff
@@ -6,13 +6,13 @@ Run edit mode by typing:
 ```
 vrecord -e
 ```
-After selecting all of your options and clicking "OK" the you will be prompted to enter a unique ID for the file. After the ID is entered, the incoming video signal will be recorded to a file with some associated metadata files. When you are done recording, close the vrecord window. If you've set a time limit for capture the vrecord window should automatically close when the time limit has been reached.
+After selecting all of your options and clicking "OK," you will be prompted to enter a unique ID for the file. After the ID is entered, the incoming video signal will be recorded to a file with some associated metadata files. When you are finished recording, you can close the vrecord window. If you've set a time limit for capture, the vrecord window will automatically close when the time limit has been reached.
```

Comment:
Is it `clicking "OK," you` in English? (In languages I know it would be `clicking "OK", you`.)

Reply:
yeah inside parenthetical (to my knowledge)